src/grt/src/fastroute/src/FastRoute.cpp

```diff
@@ -682,6 +684,98 @@ NetRouteMap FastRouteCore::getRoutes()
   return routes;
 }
 
+NetRouteMap FastRouteCore::getPartialRoutes()
```

Comment:
What is the definition of a partial route?

Reply:
I think partial route was a bad naming, but it is the routes before layer assignment.

Reply:
I was trying to make it possible to use both before and after layer assignment, then it could be just one function with getRoutes(),  if this makes more sense I can do it.

Reply:
If it can convert both sorts of routes that is good.  If not we should rename it.

Reply:
Oh it seems it wasn’t pushed, but I made both methods into one. I will push it.

Reply:
I tried to merge this method with the method getRoutes(), but this caused some errors in other parts of the flow. So this method was renamed to getPlanarRoutes(), for better understanding of it's purpose: generate the routes before layer assignment.